platforms/jvm/plugins-application/src/main/java/org/gradle/jvm/application/tasks/CreateStartScripts.java

```diff
@@ -165,9 +165,10 @@ public String getOptsEnvironmentVar() {
     /**
      * The environment variable to use to control exit value (Windows only).
      */
-    @Nullable
-    @Optional
     @Input
+    @Optional
+    @Nullable
+    @Deprecated
```

Comment:
Like mentioned in the prior comment, this method (and the below setter) will need deprecation nagging added.

However, since this property is an `@Input`, Gradle will automatically call this method even though there is no reason for it to. This will emit a deprecation warning even if the user does not call the method themselves.

Instead, we should mark this method as `@Internal` so that Gradle does not try to fetch the value when calculating the fingerprint for this task. We can do this since the property value no longer affects task output.

Reply:
Marked as `@Internal`, which in turn forced me to remove `@ToBeReplacedByLazyProperty`.